hypha/apply/users/views.py

```diff
@@ -159,7 +159,10 @@ class EmailChangeDoneView(TemplateView):
 
 @login_required()
 def become(request):
-    if not request.user.is_apply_staff:
+    if not settings.HIJACK_ENABLE:
+        raise PermissionDenied()
```

Comment:
In theory we should return 404 here. Permission denied is dependent on user and when hijack is disabled it would not work for any user.

..the reason I wanted the urls to not exist when hijack is disabled. But we could also go-ahead with this way.

Reply:
Added `HIJACK_ENABLE = True` in test.py, then it work to conditional add the become path. Looks ok?